SalesforceCommerceCloud/pwa-kit
# Release 2.7: fix order of hooks

Return early only after all of the hooks are called, as per [React doc](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level).

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

1. Navigate to any PDP page
2. Add the item to cart
3. A modal would pop up upon successful add
3. In the browser console, verify that there's no more error message from React re: order of hooks

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
